mintpy/utils/utils1.py

```diff
@@ -830,6 +830,24 @@ def run_deramp(fname, ramp_type, mask_file=None, out_file=None, datasetName=None
             prog_bar.close()
             print('finished writing to file: {}'.format(fname))
 
+    elif k == 'velocity':
+        if not datasetName:
+            datasetName = 'velocity'
+        if coeff_file:
+            # prepend file-type to line of coefficients
+            with open(coeff_file, 'a') as f:
+                f.write('{}    '.format(atr['FILE_TYPE']))
+        # read
+        data = readfile.read(fname, datasetName=datasetName)[0]
+        # deramp
+        data = deramp(data, mask,
+                    ramp_type=ramp_type,
+                    metadata=atr,
+                    coeff_file=coeff_file)[0]
+        # write
+        print('writing >>> {}'.format(out_file))
+        writefile.write(data, out_file=out_file, ref_file=fname)
+
     # Single Dataset File
     else:
```

Comment:
Instead of creating a new `elif` block, we could use the existing `else` block by assigning the `datasetName` for the velocity file as below:

```python
data = readfile.read(fname)[0]
```

to

```python
if not datasetName and k == 'velocity':
    datasetName = 'velocity'
data = readfile.read(fname, datasetName=datasetName)[0]
```

Reply:
Hi @yunjunz, thanks for the advice. I will do that!